ir_cdk_stacks/in_clw_01_stack.py

```diff
@@ -0,0 +1,127 @@
+from aws_cdk import core
+from aws_cdk import aws_s3 as s3
+from aws_cdk import aws_cloudwatch as cloudwatch
+from aws_cdk import aws_events as events
+from aws_cdk import aws_lambda as _lambda
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_events_targets as event_target
+from aws_cdk import aws_sns as sns
+from aws_cdk import aws_sns_subscriptions as subs
+
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+class InClw01Stack(core.Stack):
+
+    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        NOTIFY_EMAIL = self.node.try_get_context("notify_email")
+        SLACK_WEBHOOK_URL = self.node.try_get_context("webhook_url")
+        WHITE_LIST_GROUP = self.node.try_get_context("white_list_group")
+
+        if (
+                not NOTIFY_EMAIL
+                or not SLACK_WEBHOOK_URL
+                or not WHITE_LIST_GROUP
+        ):
+            logger.error(f"Required context variables for {id} were not provided!")
+        else:
+
+        # The code that defines your stack goes here
+            ep1 = {
+                "source": [
+                    "aws.logs"
+                ]
+            }
+
+            ep2 = {
+                "source": [
+                    "aws.cloudwatch"
+                ]
+            }
+
+            rule1 = events.Rule(self,
+                               "cdkRule1_clw",
+                               description='Rule created by CLW CDK',
+                               enabled=True,
+                               rule_name="rule1bycdk_clw",
+                               event_pattern=ep1)
+
+            rule2 = events.Rule(self,
+                                "cdkRule2_clw",
+                                description='Rule created by CLW CDK',
+                                enabled=True,
+                                rule_name="rule2bycdk_clw",
+                                event_pattern=ep2)
+
+            # 3. Create response lambda and add it as a target of the rule
+            lambda_dir_path = os.path.join(os.getcwd(), "ir_cdk_stacks", "in_clw_01")
+            response_lambda = _lambda.Function(
+                self,
+                "InClw01ResponseFunction",
+                runtime=_lambda.Runtime.PYTHON_3_7,
+                handler="clwUnauthAccessResponse.lambda_handler",
+                code=_lambda.Code.from_asset(lambda_dir_path),
+                function_name="InClw01ResponseFunction",
+                environment = {
```

Comment:
Inject your created policies here so you don't have to hardcode the PolicyName in the lambda handler.

Reply:
I understand your point but it will still remain dynamic if I don't change anything.